frontend/src/components/NotesPageLoggedOutView.tsx

```diff
@@ -0,0 +1,5 @@
+const TrashPageLoggedOutView = () => {
```

Comment:
This shouldn't be a view or page, this should be a generic component stating something generic about requiring a log in and we should be able to use it everywhere we need an account but don't have one. 

The copy should be something like: `You need to log in to see this page`

Reply:
Otherwise you're gonna repeat this pattern for every page you implement in the future. Please revert this.